eclair-core/src/main/scala/fr/acinq/eclair/router/Router.scala

```diff
@@ -122,12 +122,12 @@ class Router(val nodeParams: NodeParams, watcher: typed.ActorRef[ZmqWatcher.Comm
     case Event(SyncProgress(progress), d: Data) =>
       Metrics.SyncProgress.withoutTags().update(100 * progress)
       if (progress == 1.0 && d.channels.nonEmpty) {
-        log.info("initial routing sync done")
+        log.debug("initial routing sync done")
```

Comment:
```suggestion
        log.info("initial routing sync done")
```

Reply:
Reverted in https://github.com/ACINQ/eclair/pull/2733/commits/763720f8ac473768685438d3edf55c0d5b1bc371